runtime/src/lib.rs

```diff
@@ -124,7 +124,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // The version of the runtime specification. A full node will not attempt to use its native
     //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
-    spec_version: 200,
+    spec_version: 210,
```

Comment:
210 is on master now. Maybe 220, for example?

Reply:
Yeah, already isn't actual value